app/src/main/java/fi/helsinki/opintoni/service/CalendarTransactionalService.java

```diff
@@ -53,7 +54,7 @@ public CalendarFeedDto getCalendarFeed(Long userId) {
 
     public CalendarFeedDto createCalendarFeed(Long userId) {
         CalendarFeed calendarFeed = new CalendarFeed();
-        calendarFeed.user = userRepository.findOne(userId);
+        calendarFeed.user = userRepository.findById(userId).orElseThrow(() -> new NotFoundException(""));
```

Comment:
Voisi tehdä tuolle `NotFoundException`ille rakentajan joka ei vaadi parametria (mikä asettaisi stringiksi geneerisen "Not found"-tekstin), jolloin tässä (ja muualla) voisi sanoa vain `NotFoundException::new`.